libqfieldsync/layer.py

```diff
@@ -136,6 +136,8 @@ def __init__(self, layer):
         self._photo_naming = {}
         self._relationship_maximum_visible = {}
         self._is_geometry_locked = None
+        self._is_geometry_locked_expression_active = False
+        self._is_geometry_locked_expression = ""
```

Comment:
```suggestion
        self._geometry_locked_expression = ""
```
Does not make sense to have the `is_` prefix for non-boolean expression.

Reply:
Done